package/MDAnalysis/version.py

```diff
@@ -67,4 +67,4 @@
 # e.g. with lib.log
 
 #: Release of MDAnalysis as a string, using `semantic versioning`_.
-__version__ = "0.20.2-dev0"  # NOTE: keep in sync with RELEASE in setup.py
+__version__ = "1.0.0-dev0"  # NOTE: keep in sync with RELEASE in setup.py
```

Comment:
Tests are failing because this tag is different from the one in `MDAnalysisTests/__init__.py`.

```
E       AssertionError: MDAnalysis release 1.0.1-dev0 must be installed to have meaningful tests, not 1.0.0-dev0
E       assert '1.0.0-dev0' == '1.0.1-dev0'
E         - 1.0.1-dev0
E         ?     ^
E         + 1.0.0-dev0
E         ?     ^

```